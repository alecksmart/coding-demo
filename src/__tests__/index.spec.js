/* eslint-disable import/order */
/* eslint-disable import/no-unresolved */
import React from 'react';
import { Provider } from 'react-redux';
import store from '../store';
import shallowWithStore from 'shallowWithStore';
import GameBoard from '../modules/GameBoard/index';

describe('>>> Client entry point', () => {
  it('+++ should capture the snapshot of GameBoard', () => {
    const renderedValue = shallowWithStore(
      <Provider store={store}>
        <GameBoard />
      </Provider>,
      store,
    );
    expect(renderedValue).toMatchSnapshot();
  });
});
