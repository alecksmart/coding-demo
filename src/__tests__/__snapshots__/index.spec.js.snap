// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> Client entry point +++ should capture the snapshot of GameBoard 1`] = `ShallowWrapper {}`;
